setup.py

```diff
@@ -20,11 +20,16 @@
 
 from __future__ import annotations
 
-from core import feconf
+# The third party library path is defined in common.py,
+# which is required to import typing_extensions.
+from scripts import common # pylint: disable=unused-import
 
 import pkg_resources
 import setuptools
 
+# Since feconf imports typing_extensions, it should be
+# imported after common is imported.
+from core import feconf # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
```

Comment:
Can you move this directly below the common import?

Reply:
I had to move the common import below. The linter fails if I do it the other way round because of `isort:skip`.